src/config/featured.json

```
@@ -14,6 +14,13 @@
       "description": "KNIX MicroFunctions is a serverless computing platform that combines container-based resource isolation with a lightweight execution model using processes to significantly improve resource efficiency and decrease the function startup latency. KNIX MicroFunctions works in Knative as well as bare metal or virtual machine-based environments.",
       "logo": "https://raw.githubusercontent.com/knix-microfunctions/knix/develop/logo/KNIX_logo_stack.jpg"
     },
+    {
+      "name": "NTT",
```

We are still not 100% sure if it's "NTT" or "ntt", but the trend goes:
```suggestion
      "name": "ntt",
```